src/extensions/paramHelper.ts

```diff
@@ -16,6 +16,41 @@ import * as askUserForParam from "./askUserForParam";
 import * as track from "../track";
 import * as env from "../functions/env";
 
+/**
+ * Interface for holding different param values for different environments/configs.
+ */
+export interface ParamBindingOptions {
+  default: string;
+  local?: string;
+  // Add project specific key:value here when we want to support that.
+}
+
+export function getDefaultParamBindings(params: { [key: string]: ParamBindingOptions }): {
+  [key: string]: string;
+} {
+  let ret = {};
+  Object.entries(params).forEach(([k, v]) => {
```

Comment:
Prefer `for... of...` instead of `forEach()`  - [go/tsjs-practices/iteration](https://goto.google.com/tsjs-practices/iteration)